webviz_subsurface/plugins/_reservoir_simulation_timeseries_regional.py

```diff
@@ -153,43 +153,41 @@ def __init__(
         self.field_totals = [
             col for col in self.smry.columns if fnmatch.fnmatch(col, "F[OWG]PT")
         ]
+        self.rec_ensembles = set()
         if self.field_totals:
-            self.smry_init_prod = pd.concat(
+            smry_init_prod = pd.concat(
                 [
                     df[["ENSEMBLE", "DATE"] + self.field_totals][
                         df["DATE"] == min(df["DATE"])
                     ]
                     for _, df in self.smry.groupby("ENSEMBLE")
```

Comment:
Filter first, then groupby, to minimize copy size
```suggestion
                    df[df["DATE"] == min(df["DATE"])]
                    for _, df in self.smry[["ENSEMBLE", "DATE"] + self.field_totals].groupby("ENSEMBLE")
```